scripts/setup

```diff
@@ -7,4 +7,8 @@ rbenv install -s
 gem install bundler
 bundle install
 rm -rf .git/hooks/pre-commit
-cp -f scripts/clang_format_git_diff .git/hooks/pre-commit
\ No newline at end of file
+cp -f scripts/clang_format_git_diff .git/hooks/pre-commit
+
+/usr/local/bin/carthage bootstrap --platform iOS
+
+plutil -replace CFBundleVersion -string 4.0.0 Carthage/Build/iOS/SDWebImage.framework/Info.plist
```

Comment:
SDWebImage is lacking a CFBundleVersion which causes a failure when trying to submit the app to the store. This fix-up could be done only on archive, but I also figured I'd eventually want to add something to fix the build warnings that the frameworks are currently generating.